### PR TITLE
Added note about Drush9 and Xdebug

### LINKED ISSUE #786: Drush 9 path mappings?
Version info:

- cli 2.2-7.1
- Drush 9.3.0

I'm running into issues with getting Xdebug to work with Drush 9 and Docksal.

I have the following path mappings:

repo root = /Users/les.peabody/Sites/corvette

- repo root -> /var/www
- repo root/docroot -> /var/www/docroot
- repo root/vendor/bin/drush -> /usr/local/bin/drush

I'm pretty confident I've precisely followed the instructions for setting up Drush debugging https://docksal.readthedocs.io/en/master/tools/xdebug/.

Ultimately, no Drush script or command can be caught with a breakpoint. I checked off "Break at first line of script" and stepped in until I got a path mappings error, and I get the following error:

```
Cannot find a local copy of the file on server /usr/local/bin/drush/bin/drush.php
Local path is /Users/les.peabody/Sites/corvette/vendor/bin/drush/bin/drush.php
```

Maybe it's the end of the day, but for whatever reason I can't figure our how to properly set the path mappings to get everything connected. If you could offer any insight I would be most appreciative. Thanks!

### PR DESCRIPTION
**Pull request by sean-e-dietrich**
Added note that Drush Launcher will not work with XDebug.

Fixes #786 

Also removed extra config in the section for vscode as this is part of variables already baked in.

### CODE REVIEW

File: docs/content/tools/xdebug.md [hunk 1]
```diff
@@ -95,6 +95,13 @@ Note: If `PHP_IDE_CONFIG=serverName=${VIRTUAL_HOST}` is set before web request d
 
 You can run your scripts in console and debug them in the same way as browser requests. For example, you can run drush command: `fin drush fl` and debug this drush command from the Features module.
 
+
+### Drush 9
+
+In order to use Drush 9 with XDebug you will need to run commands directly through the `exec` command to the bin directory where Drush is located.
+This can usually be done by running `fin exec -T vendor/bin/drush` from where the vendor directory is located instead of running the regular command
```
Comment (achekulaev) [inline docs/content/tools/xdebug.md]: Why the -T ?

Comment (sean-e-dietrich) [reply docs/content/tools/xdebug.md]: This should have been removed some reason I thought the existing Drush command had the -T command.

File: docs/content/tools/xdebug.md [hunk 2]
```diff
@@ -205,8 +212,6 @@ version: "2.1"
 services:
   cli:
     environment:
-      - XDEBUG_ENABLED=1
```
Comment (achekulaev) [inline docs/content/tools/xdebug.md]: Hey, so why is this removed here. I see the line above says `Ensure that your docksal.yml or docksal-local.yml has been updated to include `XDEBUG_ENABLED=1` in the CLI` but here you removed these 2 lines, why so?

Comment (sean-e-dietrich) [reply docs/content/tools/xdebug.md]: this is controlled by the `XDEBUG_ENABLED` variable and is redundant in my opinion.
